From fdf82a34f425fbf03ab19993500340d146fe1643 Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@freescale.com>
Date: Thu, 29 Aug 2013 17:31:41 -0300
Subject: [PATCH 0529/1072] ASoC: fsl: Drop SND_SOC_FSL_UTILS from
 SND_SOC_IMX_SPDIF

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit def22a96dec1485df9f4d3bccd05e4bc886180b2

SND_SOC_FSL_UTILS is only used by PowerPC machines, so let's drop it in the
i.mx case.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Acked-by: Nicolin Chen <b42378@freescale.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 sound/soc/fsl/Kconfig |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 8d1822d..e3d3e01 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -225,7 +225,6 @@ config SND_SOC_IMX_SPDIF
 	tristate "SoC Audio support for i.MX boards with S/PDIF"
 	select SND_SOC_IMX_PCM_DMA
 	select SND_SOC_FSL_SPDIF
-	select SND_SOC_FSL_UTILS
 	select SND_SOC_SPDIF
 	help
 	  SoC Audio support for i.MX boards with S/PDIF
-- 
1.7.5.4

